### PR TITLE
Bulgarian index.js

### PR DESCRIPTION
**Pull request by imbyurhan**

### CODE REVIEW

File: store/bg/index.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+export const store = new Vuex.Store({
+  state: {
+    selectedMethod: '',
+    adding: [
+      {
+        name: 'splice',
+        shortDesc: 'елемент(и) в масив',
+        desc: 'Добавя и/или премахва елементи от масив.',
+        example: `arr.splice(2, 0, 'tacos');<br>
+        console.log(arr);`,
```
Comment (madhusudhan1234) [inline store/bg/index.js]: Please remove console.log :)

Comment (Gonzalo2310) [reply store/bg/index.js]: In this case console.log is not code is text to display
captura de pantalla 2017-12-29 a las 17 34 03

Comment (Gonzalo2310) [reply store/bg/index.js]: For this to be complete it must also go even if it is the same in all languages.

Comment (madhusudhan1234) [reply store/bg/index.js]: I understand the mechanism, Thank you @Gonzalo2310 .

**Comment by sdras** [comment]
Thank you so much! In order to merge and have a full Bulgarian translation, we just need a `bg` directory/version here, for the dropdown components: [https://github.com/sdras/array-explorer/tree/master/src/locale]( https://github.com/sdras/array-explorer/tree/master/src/locale)

🎉

**Comment by imbyurhan** [comment]
Added a Bulgarian version for dropdown components.

**Comment by sdras** [comment]
You. Are. Awesome!! Thank you so much 🎉

Just need a couple more PRs for some of the other languages that are almost done and I'll rebuild the site. Your help is really appreciated!

**Pull request merged**